src/app/profile/page.tsx

```diff
@@ -59,51 +76,133 @@ export default function ProfilePage() {
   const handleSubmit = async (e: any) => {
     e.preventDefault();
 
-    const newErrors: any = {};
-    if (!formData.name.trim()) newErrors.name = "Name is required.";
-    if (!formData.email.trim()) newErrors.email = "Email is required.";
+    if (!editMode) {
+      setEditMode(true);
+    } else {
+      const newErrors: any = {};
+      if (!formData.name.trim()) {
+        newErrors.name = "Name is required.";
+      }
+      if (!formData.email.trim()) {
+        newErrors.email = "Email is required.";
+      }
+      if (!formData.phone.trim()) {
+        newErrors.phone = "Phone is required.";
+      }
+      if (!formData.city.trim()) {
+        newErrors.city = "City is required.";
```

Comment:
So this is what the user will see on their profile the first time after a Google Sign Up, right?